.circleci/src/commands/lint.yml

```diff
@@ -6,5 +6,8 @@ steps:
   - run:
       name: Linting check
       command: yarn workspace << parameters.directory >> run lint:ci
+  # - run: TODO
```

Comment:
Should be enabled before merging together with a PR applying all new prettier rules.

Reply:
done in second PR